.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js: 8
+dist: trusty
+sudo: required
+git:
+  depth: false
+branches:
+  only:
+    - experimental/travis
```

Comment:
Should this be _all`?

Reply:
Not yet! Just want to test this branch for now. It'll get changed to "all" once this is stable.